src/combinatorics/adj_matrix.lean

```diff
@@ -0,0 +1,88 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author:  Aaron Anderson, Jalex Stark.
+-/
+import linear_algebra.matrix
+import data.rel
+import combinatorics.simple_graph
+
```

Comment:
Could you add a module doc, similar to `combinatorics.simple_graph`'s?

At the moment, the word "adjacency" is not mentioned anywhere in this file, making it hard to look it up.